client/app/components/datagrid/OperationDataGrid.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
```

Comment:
Well I definitely think that this code needed to be moved out of the `DataGrid` component. It's best to keep components isolated and simplified, especially something like `DataGrid` which is used in multiple places. 

That said we could likely get rid of `OperationDataGrid` and `OperatorsDataGrid` and move the code in these one file up to `AccessRequests.tsx` and  `Operations.tsx`. Let me know what you think about that.

Reply:
I remembered the reason I added these to their own component instead of their parent components (`AccessRequests.tsx` and `Operations.tsx`). Since the parent components are server components I got an error when trying to pass the `fetchPagedata` action handlers so I needed a client component to pass these in.